### PR TITLE
feat: update oracle skills and demo

### PR DESCRIPTION
**Pull request by jrriehl**
## Proposed changes

Simplifies oracle skills by removing the libp2p connection and defaulting to the `fetchai` ledger.

## Types of changes

What types of changes does your code introduce to agents-aea?
_Put an `x` in the boxes that apply_

- [ ] Bugfix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to not work as expected)

## Checklist

_Put an `x` in the boxes that apply._

- [x] I have read the [CONTRIBUTING](https://github.com/fetchai/agents-aea/blob/main/CONTRIBUTING.md) doc
- [x] I am making a pull request against the `develop` branch (left side). Also you should start your branch off our `develop`.
- [x] Lint and unit tests pass locally with my changes and CI passes too
- [x] I have added tests that prove my fix is effective or that my feature works
- [x] I have checked that code coverage does not decrease.
- [x] I have added necessary documentation (if appropriate)
- [x] Any dependent changes have been merged and published in downstream modules

### CODE REVIEW

**Review by solarw — APPROVED** [review]
LGTM

**Review by 5A11 — APPROVED** [review]

**Review by 5A11 — APPROVED** [review]

**Comment by 5A11** [comment]
@solarw please resolve the merge conflicts.

File: packages/fetchai/agents/coin_price_oracle/aea-config.yaml [hunk 1]
```diff
@@ -9,7 +9,10 @@ fingerprint_ignore_patterns: []
 connections:
 - fetchai/http_client:0.24.1
 - fetchai/ledger:0.21.0
+<<<<<<< HEAD
```
[the remark this answers is not in the document]
Comment (5A11) [reply packages/fetchai/agents/coin_price_oracle/aea-config.yaml:12]: and fix the ones below

File: docs/oracle-demo.md [hunk 2]
```diff
@@ -35,17 +34,24 @@ aea create coin_price_oracle
 cd coin_price_oracle
 aea add connection fetchai/http_client:0.24.1
 aea add connection fetchai/ledger:0.21.0
+<<<<<<< HEAD
```
[the remark this answers is not in the document]
Comment (solarw) [reply docs/oracle-demo.md]: fixed

File: docs/oracle-demo.md [hunk 3]
```diff
@@ -35,17 +34,24 @@ aea create coin_price_oracle
...
 aea add connection fetchai/p2p_libp2p:0.27.1
+>>>>>>> 5ae92cc932570478f7540053275cb71e014ad013
 aea add skill fetchai/advanced_data_request:0.7.1
 aea add skill fetchai/simple_oracle:0.16.1
 aea config set --type dict agent.dependencies \
 '{
   "aea-ledger-fetchai": {"version": "<2.0.0,>=1.0.0"},
   "aea-ledger-ethereum": {"version": "<2.0.0,>=1.0.0"}
 }'
+<<<<<<< HEAD
```
[the remark this answers is not in the document]
Comment (solarw) [reply docs/oracle-demo.md]: fixed

File: packages/fetchai/agents/coin_price_oracle/aea-config.yaml (hunk 1 above)
[the remark this answers is not in the document]
Comment (solarw) [reply packages/fetchai/agents/coin_price_oracle/aea-config.yaml]: fixed

**Comment by 5A11** [comment]
Failure for test_simple_oracle.py. I think it is minor and just needs an update to a package number (`0.17.0` to `0.17.1`).

**Review by 5A11 — APPROVED** [review]
lgtm

**Pull request merged**